lib/package/policySharedFlow.js

```diff
@@ -0,0 +1,238 @@
+//PolicySharedFlow.js
```

Comment:
copied from Policy.js, as mentioned previously it is better to reuse existing one